packages/@o3r/schematics/src/utility/matching-peers.ts

```diff
@@ -1,6 +1,18 @@
 import * as fs from 'node:fs';
 import { PackageJson } from 'type-fest';
 
+
+/**
+ * Retrieve the peer dependency version for a package
+ *
+ * @param packageJsonPath
+ * @param packageName
+ */
+export function getPeerDepVersion(packageJsonPath: string, packageName: string) {
+  const packageJsonContent: PackageJson = JSON.parse(fs.readFileSync(packageJsonPath, { encoding: 'utf-8' }));
```

Comment:
Not working with Tree which should be avoid as much as possible in this package that supposed to provide helpers for schematics.
(It is not the only helper using `fs` (or `glob`) but we really need to care of it).

Reply:
Ok but in that case it can be worth to put some words in the comment regarding the fact that this functions is accessing directly to the FS (bypass tree mechanism).